### DISCUSSION TITLE
Google Colab

### DISCUSSION THREAD

**Discussion by Mario03482** [AndreDalwin/Whisper2Summarize#1] [Ideas]
Can you create a colab with this repo

**Comment by AndreDalwin** [comment]
Just made one! You may access the notebook here: [Whisper2Summarize Colab Edition](https://colab.research.google.com/github/AndreDalwin/Whisper2Summarize/blob/main/Whisper2Summarize_Colab_Edition.ipynb)

Updated the README as well on how to use this.
